src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-storage-class.model.ts

```diff
@@ -104,3 +86,30 @@ export interface PlacementTarget {
 export const CLOUD_TIER = 'cloud-s3';
 
 export const DEFAULT_PLACEMENT = 'default-placement';
+
+export const ALLOW_READ_THROUGH_TEXT =
+  'Enables fetching objects from remote cloud S3 if not found locally (default: false).';
+
+export const MULTIPART_MIN_PART_TEXT =
+  'It specifies that objects this size or larger are transitioned to the cloud using multipart upload.';
+
+export const MULTIPART_SYNC_THRESHOLD_TEXT =
+  'It specifies the minimum part size to use when transitioning objects using multipart upload.';
+
+export const TARGET_PATH_TEXT =
+  'Target Path refers to the storage location (e.g., bucket or container) in the cloud where data will be stored.';
+
+export const TARGET_REGION_TEXT =
+  'The region of the remote cloud service where storage is located.';
+
+export const TARGET_ENDPOINT_TEXT =
+  'The URL endpoint of the remote cloud service for accessing storage.';
+
+export const TARGET_ACCESS_KEY_TEXT =
+  "To view or copy your access key, go to your cloud service's user management or credentials section, find your user profile, and locate the access key. You can view and copy the key by following the instructions provided.";
+
+export const TARGET_SECRET_KEY_TEXT =
+  "To view or copy your secret key, go to your cloud service's user management or credentials section, find your user profile, and locate the secret key. You can view and copy the key by following the instructions provided.";
+
+export const RETAIN_HEAD_OBJECT_TEXT =
+  'Retain object metadata after transition to the cloud (default: false).';
```

Comment:
Looks like we are enabling it by default but setting default as false in  helper text.
```suggestion
  'Retain object metadata after transition to the cloud.';
```